adding_species_manuscript.tex

```diff
@@ -453,34 +421,65 @@ \subsection*{Additions to \stdpopsim}
 a nematode (\textit{Caenorhabditis elegans}),
 two flowering plants including a crop (\textit{Helianthus annuus}),
 an algae (\textit{Chlamydomonas reinhardtii}),
-and two bacteria,
-in addition to four primates 
-and a common mammalian associate of primates (\textit{Canis familiaris}). % Elise: An aside, I find this way of describing dogs to be hilariously academic
-Not all of these have genetic maps or demographic models,
+two bacteria,
+%in addition to
+four primates and a common mammalian associate of primates (\textit{Canis familiaris}).
+% Elise: An aside, I find this way of describing dogs to be hilariously academic
+% Ilan: I concur :-)
+Not all of these have genetic maps or demographic models (see Figure \ref{fig:tree}),
 but this lays the framework for future contributions.
+\begin{figure}
+    \includegraphics[width=\linewidth]{figs/species_fig}
+	\caption{Phylogenetic tree of species available in the \stdpopsim catalog,
+		including the six species we published in the original release \citep[in blue]{Adrion2020}, and 15 species that have since been added (in orange).
+		Solid circles indicate species that have one (light grey) or more
+		(dark grey) demographic models and genetic maps.
+	\label{fig:tree} }
+\end{figure}
 
-A key feature added to \stdpopsim that expands the diversity of species that can
-be realistically modeled is the inclusion of recombination by gene conversion,
-which is essential for organisms such as \textit{E.\ coli}
-that lack crossing over.
-\colorbox{yellow}{[ TODO: summarize model of GC (see PR\#57) ]}
+Expanding the species catalog required adding several functionalities to the simulation framework.
+We thus upgraded the neutral simulation engine, \texttt{msprime}, from version 0.7.4 to version 1.0. \citep{Baumdicker2022}.
+This upgrade provides a number of benefits such as a discrete site model of mutation,
+so that simulated data will now %, as in real data,
+have a small proportion of sites with multiple mutations and possibly more than two alleles.
+Another key feature enabled by this upgrade was recombination by gene conversion,
+which is essential for modeling genomes of bacteria and archea.
+Gene conversion affects shorter segments than crossover recombination and creates distinct patterns of genetic diversity along the genome~\citep{Korunes2017}.
+In bacteria and archaea, homologous recombination occurs primarily through gene conversion rather than single crossover recombinations.
+As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers.
+Gene conversion in such species is implemented in \stdpopsim by setting the
+\texttt{bacterial\_recombination} flag to \texttt{True} (as a feature of the genome)
+and the  \texttt{gene\_conversion\_length} parameter to the average gene conversion tract length.
```

Comment:
I don't think we need to put in the actual API options?

Reply:
Hm, also - the gene conversion option works in SLiM now also (where the tract length is not geometric but negative binomial), but this paragraph is headed by 'msprime'.

Reply:
I'll remove the mention of specific flags and parameter names, but will probably still say that there is a flag that tells the engine to use a model of non-crossover recombination. I'm also differentiating this discussion from the upgrade of msprime to v1.0 (but keeping it in the same paragraph ) and explicitly mentioning SLiM to clarify that the same model can be simulated with both engines